src/main/java/net/minecraftforge/fluids/BlockFluidBase.java

```diff
@@ -728,4 +728,21 @@ public AxisAlignedBB getCollisionBoundingBox(@Nonnull IBlockState blockState, @N
     {
         return NULL_AABB;
     }
+    
+    @SideOnly (Side.CLIENT)
```

Comment:
stick an @Override here in case the signature changes in the future

Reply:
Added override annotation as requested.